hooks/post/30-upgrade.rb

```diff
@@ -66,6 +66,15 @@ def remove_gutterball
   end
 end
 
+def remove_event_queue
+  queue_present = `qpid-stat -q --ssl-certificate=/etc/pki/katello/qpid_client_striped.crt -b amqps://localhost:5671 | grep :event | wc -l`.chomp.to_i
```

Comment:
Doesn't Kafo migrations have the answer files? While currently I don't think these paths are exposed as parameters now, I wonder if qpid will move to a top level module.

I don't think this is very important right now, but I wanted to raise the question.